tests/test_extension.py

```diff
@@ -9,25 +10,55 @@ def test_docstring():
     :docstring:
 """
     output = markdown.markdown(content, extensions=["mkautodoc"])
-    assert output.splitlines() == [
-        "<h1>Example</h1>",
-        '<div class="autodoc">',
-        '<div class="autodoc-signature"><code>mocklib.<strong>example_function</strong></code><span class="autodoc-punctuation">(</span><em class="autodoc-param">a</em><span class="autodoc-punctuation">, </span><em class="autodoc-param">b=None</em><span class="autodoc-punctuation">, </span><em class="autodoc-param">*args</em><span class="autodoc-punctuation">, </span><em class="autodoc-param">**kwargs</em><span class="autodoc-punctuation">)</span></div>',
-        '<div class="autodoc-docstring"><p>This is a function with a <em>docstring</em>.</p></div>',
-        "</div>",
-    ]
+    assert_xml_equal(
+        output,
+        """\
```

Comment:
Is the backslash needed here, and why?

Reply:
The leading backslash ensures you don't have a `\n` as the first character in the string. i'm not totally sure anymore if its required for the test, but it ensures you don't get different formatting between test/expected, which can make it less confusing if you are debugging a failing test

Reply:
Okay, I reckon it'd be very marginally cleaner to drop these.
After that I think this is looking ace ✨.

Reply:
Ok, i've addressed this on #13 with this commit: https://github.com/tomchristie/mkautodoc/pull/13/commits/5536b1e312fe761942d5cc1ea82f420402575b2c

gonna close this PR since it is contained in that one.